ports/psoc6/boards/CY8CPROTO-062-4343W/mpconfigboard.h

```diff
@@ -3,3 +3,10 @@
 #define MICROPY_HW_BOARD_NAME                   "CY8CPROTO-062-4343W"
 
 #define MICROPY_PY_NETWORK_HOSTNAME_DEFAULT     "CY8C-062-4343W"
+
+#define MICROPY_GC_HEAP_SIZE                    (256 * 1024) // 256 KB
+
+#define MICROPY_PY_HASHLIB                      (1)
```

Comment:
If I get it right these are defines for mpy extended modules which means they are by default already defined (as 0?) in one of the mpy extmod files. Only if they have to be redefined to 1 , then needs to be included either common to port or board. or?

Reply:
You are right, I just have removed them from the file.

Reply:
I have only kept in the old board, and removed it in the 063-BLE.

Reply:
Perfect!